lms/static/scripts/frontend_apps/components/dashboard/test/AssignmentActivity-test.js

```diff
@@ -270,7 +270,7 @@ describe('AssignmentActivity', () => {
 
       assert.calledWith(
         fakeNavigate,
-        '?student_id=8&student_id=20&student_id=32&assignment_id=123',
+        '?assignment_id=123&student_id=8&student_id=20&student_id=32',
```

Comment:
We were previously manually building this query via `recordToQueryString`, where params keep the order they are passed to.

We now use the new filter which hardcodes the order in which params are set when present, so it's now more "opinionated".

The query string is equivalent though.